src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -17,7 +17,9 @@
 public class AddCommand extends UndoableCommand {
 
     public static final String COMMAND_WORD = "add";
+    //@@author Sheikh-Umar
```

Comment:
I don't think you need the collating twice because just one means that everything below it already belongs to you. If you want to indicate where your contribution ends, don't put your name in the second one. That said, it is not suggested to collate for one line additions.

Reply:
Noted. Will make the necessary editions.